en/xml/ha_clvm.xml

```diff
@@ -508,7 +508,7 @@ cLVM) for more information and details to integrate here - really helpful-->
    </procedure>
    <para>
     If you need further information regarding LVM2, refer to the &sls;
-    &productnumber; <citetitle>&storage;</citetitle>: <link xlink:href="dsc;/sles-12/html/SLES-all/cha-lvm.html"></link>.
+    &productnumber; <citetitle>&storage;</citetitle>: <link xlink:href="dsc;/sle-ha/12-SP5/single-html/SLE-HA-guide/#book-sleha"></link>.
```

Comment:
Please revert this change - see https://github.com/SUSE/doc-sleha/pull/199#discussion_r844098808 for reason why.